BOUNTY.md

```diff
@@ -0,0 +1,115 @@
+# GraphQL.NET Bounty Program
+
+The GraphQL.NET team has worked to create a program where community developers can work on specific new features
+of GraphQL.NET for compensation. These features are determined by GraphQL.NET maintainers to have the
+potential for high-impact improvement to the product. Design proposals that would fulfill a bounty may
+be made either by individuals or teams of developers, but the final payout will be made to only one stakeholder.
+
+
+## Bounty Process Flowchart
+
+1. The GraphQL.NET team publishes a Request For Proposals (RFP) describing a feature that the team has decided
+   should be prioritized, or a bug that is in need of being researched and fixed.
+    * Initial Bounty Status set to **Accepting Proposals**
+2. Developers write proposals for how they would implement the feature or fix the bug described in the RFP.
```

Comment:
What to do with "instant" PRs without even writing proposal about implementation or anything?

Reply:
The bounty program isn't designed for such a purpose.  We should be collaborating periodically on what we should apply a bounty to, and then create an issue for the few tasks and mark them as having a bounty.

Reply:
Then it is better to explicitly say that such PRs do not fall into the program.